private/rules/v2_lock_file.bzl

```diff
@@ -45,6 +47,27 @@ def _compute_lock_file_hash(lock_file_contents):
         to_hash.update({key: json.decode(json.encode(value))})
     return hash(repr(to_hash))
 
+def _compute_lock_file_hash(lock_file_contents):
+    lines = []
+    artifacts = _get_artifacts(lock_file_contents)
+
+    for artifact in artifacts:
+        line = "%s | %s | " % (to_external_form(artifact["coordinates"]), artifact["sha256"] if artifact["sha256"] else "")
```

Comment:
Probably want to mention that the format should also be updated in tandem with ArtifactsHash.java.

Reply:
Done. Also added a very similar comment to `ArtifactsHash.java`